sdk/attestation/Azure.Security.Attestation/README.md

```diff
@@ -117,26 +200,45 @@ var setResult = client.SetPolicy(AttestationType.SgxEnclave, policySetToken);
+IReadOnlyList<AttestationSigner> signingCertificates = (await client.GetSigningCertificatesAsync()).Value;
+```
 
-## Next steps
+## Troubleshooting
 
-* Provide a link to additional code examples, ideally to those sitting alongside the README in the package's `/samples` directory.
-* If appropriate, point users to other packages that might be useful.
-* If you think there's a good chance that developers might stumble across your package in error (because they're searching for specific functionality and mistakenly think the package provides that functionality), point them to the packages they might be looking for.
+Troubleshooting information for the MAA service can be found [here](https://docs.microsoft.com/azure/attestation/troubleshoot-guide)
+
+## Next steps
+For more information about the Microsoft Azure Attestation service, please see our [documentation page](https://docs.microsoft.com/azure/attestation/). 
 
 ## Contributing
+This project welcomes contributions and suggestions. Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution. For details, visit https://cla.microsoft.com.
+
+When you submit a pull request, a CLA-bot will automatically determine whether you need to provide a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions provided by the bot. You will only need to do this once across all repos using our CLA.
+
+This project has adopted the [Microsoft Open Source Code of Conduct][microsoft_code_of_conduct]. For more information see the Code of Conduct FAQ or contact <opencode@microsoft.com> with any additional questions or comments.
 
-This is a template, but your SDK readme should include details on how to contribute code to the repo/package.
+See [CONTRIBUTING.md][contributing] for details on building, testing, and contributing to these libraries.
 
 <!-- LINKS -->
 [style-guide-msft]: https://docs.microsoft.com/style-guide/capitalization
 [style-guide-cloud]: https://aka.ms/azsdk/cloud-style-guide
+[API_reference]: https://docs.microsoft.com/en-us/dotnet/api/azure.security.attestation?view=azure-dotnet-preview
+[azure_cli]: https://docs.microsoft.com/cli/azure
+[azure_identity]: https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/identity/Azure.Identity
+[azure_sub]: https://azure.microsoft.com/free/
+[code_of_conduct]: https://opensource.microsoft.com/codeofconduct/
+[JWK]: https://tools.ietf.org/html/rfc7517
+[base64url_encoding]: https://tools.ietf.org/html/rfc4648#section-5
+[nuget]: https://www.nuget.org/
+[DefaultAzureCredential]: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/identity/Azure.Identity/README.md#defaultazurecredential
+[contributing]: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/keyvault/CONTRIBUTING.md
```

Comment:
Did you mean to link the [root contributing guide](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md) here?   This one is specific to KeyVault.